internal/engine/bundle_builder.go

```diff
@@ -157,16 +156,16 @@ func (b *BundleBuilder) Build() (cue.Value, error) {
 	return v, nil
 }
 
-func (b *BundleBuilder) getInstanceUrl(v cue.Value) string {
+func (b *BundleBuilder) getInstanceUrl(v cue.Value) (string, error) {
 	url, _ := v.String()
 	if path := strings.TrimPrefix(url, apiv1.LocalPrefix); IsFileUrl(url) && !filepath.IsAbs(path) {
-		source := v.Pos().Filename()
-		if origin, ok := b.mapSourceToOrigin[source]; ok {
-			source = origin
+		dir, err := os.Getwd()
+		if err != nil {
+			return "", err
 		}
-		url = apiv1.LocalPrefix + filepath.Clean(filepath.Join(filepath.Dir(source), path))
+		url = apiv1.LocalPrefix + filepath.Clean(filepath.Join(dir, path))
 	}
-	return url
+	return url, nil
```

Comment:
I'm not convinced this is the right way to fix this. It feels weird to call `os.Getwd()` from inside the library. 

It seems to me the Timoni cmd package should somehow provide all the information the library needs. So, maybe `BundleBuilder` should have a new field like `cwd` pointing to the current `cwd`, instead of trying to figure it out itself.